### PR TITLE
Mobile Release v1.72.0

### PR DESCRIPTION
**Pull request by jhnstn**
## Description
  Release 1.72.0 of the react-native-editor and Gutenberg-Mobile.

  For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/4668

### CODE REVIEW

File: packages/react-native-editor/ios/Podfile.lock
Comment (hypest) [inline packages/react-native-editor/ios/Podfile.lock:309]: Not necessarily a blocker, but I wonder why this change appears in the release process PR.

Perhaps it's something overlooked in [the original PR](https://github.com/WordPress/gutenberg/pull/39430)? Just to verify, whdy @fluiddot?

**Review by hypest — APPROVED** [review]
I've left a comment [to verify the bumping of a dep](https://github.com/WordPress/gutenberg/pull/39553/files#r829924094), but otherwise this looks good to me Jason! Approving and let's wait for the rest of the PRs to be approved before starting the merge train. Thanks!

File: packages/react-native-editor/ios/Podfile.lock
Comment (fluiddot) [reply packages/react-native-editor/ios/Podfile.lock:309]: Yes, I'm afraid I overlooked updating the `Podfile.lock` file on the demo app project when applying the changes in https://github.com/WordPress/gutenberg/pull/39430, that's why it shows up here, sorry for that 🙇 .

Comment (fluiddot) [reply packages/react-native-editor/ios/Podfile.lock:309]: In any case, this shouldn't block the release PR as the change is expected.

Comment (hypest) [reply packages/react-native-editor/ios/Podfile.lock:309]: Not a prob, just making sure this is not surprising for any reason 😸👍

**Pull request merged**
